src/Microsoft.SqlTools.ServiceLayer/HostLoader.cs

```diff
@@ -27,9 +27,6 @@
 using Microsoft.SqlTools.ServiceLayer.LanguageExtensibility;
 using Microsoft.SqlTools.ServiceLayer.LanguageServices;
 using Microsoft.SqlTools.ServiceLayer.Metadata;
-#if INCLUDE_MIGRATION
```

Comment:
Remove this from the csproj as well and clean up the when condition

Reply:
You can remove the entire Choose element here apart from this one PropertyGroup element: https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj#L26

Then add `Condition="'$(BUILD_DOTNET_TOOL)' == 'true'"` to the ProertyGroup tag like we do in the Kusto csproj here: https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.Kusto.ServiceLayer/Microsoft.Kusto.ServiceLayer.csproj#L25